tests/lib/data/clientToolsCache.test.ts

```
@@ -0,0 +1,272 @@
+import type { ToolRequestBody } from "@helperai/client";
+import { db } from "@/db/client";
+import { clientToolsCache } from "@/db/schema";
+import {
+  cacheClientTools,
+  clearCachedClientTools,
+  getCachedClientTools,
+  getCachedToolsForAdmin,
+} from "@/lib/data/clientToolsCache";
+
+describe("clientToolsCache", () => {
+  let testMailbox: any;
```

ESLint

[error] 'testMailbox' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)